src/services/alert/MessageProvider/status/index.js

```diff
@@ -0,0 +1,98 @@
+const { Configuration } = require('@schul-cloud/commons');
+const api = require('../../../../helper/externalApiRequest');
+const logger = require('../../../../logger');
+
+const apiUri = Configuration.get('ALERT_STATUS_API_URL');
```

Comment:
use configuration everywhere directly. otherwise you cannot replace this value in tests dynamically.